client/src/components/AdvisorReports/FilterableAdvisorReportsTable.js

```diff
@@ -90,7 +82,7 @@ class FilterableAdvisorReportsTable extends Component {
     return weekColumns
   }
 
-  convertArrayOfObjectsToCSV(args) {
+  function convertArrayOfObjectsToCSV(args) {
```

Comment:
Maybe it is nicer to have convertArrayOfObjectsToCSV outside the component.

Reply:
Same here.